### DISCUSSION TITLE
本地运行正常，打包后找不到electron-vite内部引入的包模块

### DISCUSSION THREAD

**Discussion by hirohitosann** [alex8088/electron-vite#440] [General]
## bindings file-uri-to-path sqlstr ...等等，包太多了，安不完，而且只有打包启动才提示缺少什么包

![Snipaste_2024-03-09_15-37-28](https://github.com/alex8088/electron-vite/assets/83349389/15735414-ed94-4fae-8c99-86f086173968)

**Comment by alex8088** [comment]
https://cn.electron-vite.org/guide/troubleshooting#%E5%88%86%E5%8F%91

**Comment by hirohitosann** [comment]
## @alex8088 我使用的是cnpm，而且提示缺失的包，是我引用的包内部又引用的其他包
![Snipaste_2024-03-09_16-27-28](https://github.com/alex8088/electron-vite/assets/83349389/33900068-2bb9-4232-bf62-5824ca90e2ea)
![Snipaste_2024-03-09_16-27-58](https://github.com/alex8088/electron-vite/assets/83349389/79a1dd43-b57b-4b8c-ad85-411db07d3e7f)

**Comment by alex8088** [comment]
不能用cnpm，请改用npm

**Comment by hirohitosann** [comment]
## 还是不行，换node版本，npm，pnpm，yarn都不行，预安装的时候就卡死了，cnpm 是能把包安上，但打包就出问题了。不支持sqlite和mysql2
![微信图片_20240309193849](https://github.com/alex8088/electron-vite/assets/83349389/166315dd-cfae-471e-a4ff-2496c3c2c0fc)

**Comment by alex8088** [comment]
不能cnpm，cnpm都是link方式，改用其他包管理器，网路问题

**Comment by alex8088** [comment]
原生模块都需要根据electron版本重新编译，你的环境不支持，自行检索

**Comment by hirohitosann** [comment]
替换为 yarn即可
